pkg/apis/build/v1alpha1/buildstrategy.go

```diff
@@ -59,6 +68,31 @@ type Parameter struct {
 	Defaults *[]string `json:"defaults"`
 }
 
+// BuildStrategyVolume is a volume that will be mounted in build pod during build step
+// of the Build Strategy
+type BuildStrategyVolume struct {
```

Comment:
One other thing we can do - inline the `BuildVolume` struct:

```suggestion
// BuildStrategyVolume is a volume that will be mounted in build pod during build step
// of the Build Strategy
type BuildStrategyVolume struct {
    BuildVolume `json:",inline"`
    
    Overridable *bool `json:"overridable,omitempty"`
```